style.css

```diff
@@ -0,0 +1,59 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap');
+
+body {
+  font-family: 'Poppins', sans-serif;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  margin: 26px 25px 0 25px;
+}
+
+#menue-button {
+  padding-top: 7px;
+  padding-bottom: 7px;
+}
+
+#logo {
+  color: #6070ff;
+  font-weight: 700;
+  font-size: 18px;
+}
+
+#intro {
+  background: url("./images/header-shapes-mobile.jpg") no-repeat;
```

Comment:
- [ ] [OPTIONAL] At the moment, the background image is not properly positioned to match the design. Please adjust it so it looks like the original design